### DISCUSSION TITLE
Unable to uninstall WebView2 runtime silently

### DISCUSSION THREAD

**Discussion by anique9999** [MicrosoftEdge/WebView2Feedback#3950] [Q&A]
Hi,

When I'm trying to uninstall WebView2 runtime silently using these command line :- 

Path:- `%ProgramFiles(x86)%\Microsoft\EdgeWebView\Application\99.0.1150.55\Installer\setup.exe`
command:- `--force-uninstall --uninstall --msedgewebview --system-level`

But it is giving me this error `Command exited with unmapped or failure return code 19`

can someone please tell why it giving me this error and how to solve it?

**Comment by tonyhallett** [comment]
Getting the same exit code when using Process class in C# and the SilentUninstallCommand from the registry.  Is this a success exit code ( given that it succeeds in uinstalling ) ?

Using command prompt no problems.

**Comment by OutsiderLost** [comment]
It didn't work for me either, only if I also deleted EDGE. (The system just sucks, with a lot of unnecessary settings, don't must EDGE).

**Comment by skranz1982** [comment]
I get the same exit code when uninstalling v104.xxx or v105.xxx using "setup.exe --uninstall --msedgewebview --system-level --verbose-logging". Interestingly, everything looks good afterwards: files are gone, registry key is gone. Is "19" maybe a message like "everything's fine, I just need a reboot" or something similar?

**Comment by tonyhallett** [comment]
@victorhuangwq Seems like an issue to me

**Reply by victorhuangwq** [reply]
will convert to issue if it's not expected.

**Comment by victorhuangwq** [comment]
@lflores-ms could you take a look at this question?

**Comment by falsneg** [comment]
exit code 19 in that context means successfully uninstalled; see https://chromium.googlesource.com/chromium/src/+/c38084e12ff9707e2cc07ff82e004980c43c99c9/chrome/installer/util/util_constants.h
